salt/fileserver/roots.py

```diff
@@ -179,25 +177,25 @@ def update():
                         mtime_map_path,
                         line,
                     )
+    except (IOError, OSError):
+        pass
```

Comment:
Should we pass here, or log a warning or something?

Reply:
This is just an EAFP change which forgoes stat-ing the file via `if os.path.exists()`. There was no log message before when the file didn't exist, and it's not an error condition, so I'm not sure what benefit adding a log message here would bring.

Reply:
👍 Seems fine then. I just get paranoid when I see silent exception handling because I have been burned too many times by errors passing silently. The number of days I've wasted hunting down problems for that reason 😬